crmsh/bootstrap.py

```diff
@@ -1134,22 +1133,59 @@ def init_ssh():
     Configure passwordless SSH.
     """
     utils.start_service("sshd.service", enable=True)
-    configure_local_ssh_key()
+    for user in ["root", "hacluster"]:
+        configure_local_ssh_key(user)
 
 
-def configure_local_ssh_key():
+def key_files(user):
+    """
+    Find home directory for user and return key files with abspath
+    """
+    home_dir = userdir.gethomedir(user)
+    private_key = "{}/.ssh/id_rsa".format(home_dir)
+    public_key = "{}/.ssh/id_rsa.pub".format(home_dir)
+    authorized_file = "{}/.ssh/authorized_keys".format(home_dir)
+    return private_key, public_key, authorized_file
```

Comment:
I wouldn't return a 3 elements list here. This is a potential issue if for some reason you check the return data, as you will need to change all the places where the method is used.
Better to return a dictionary or specific object, so you can use just the needed elements

Reply:
yop valid point

Reply:
Thanks @arbulu89 , already changed!